site/includes/layouts/page.njk

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 ---
-<div id="container" class="py-10 w-full">
+<article id="container" class="py-10 w-full">
```

Comment:
Our CSS selector that styles the `a` links (line 13 in `styles/tailwind.css` assumes the links are inside an `article` element, which was missing from the `includes/layouts/page.njk` layout, or `page` layout.

Layouts are specified in the front matter of any .md, .njk, or .html page in site/. We currently have the following layouts:
`docs`, which gives the page that table of contents sidebar on the docs pages and applied automatically to any .html placed in site/docs/*
`page`, used for any generic page
`post`, used for any post and applied automatically to any .md file placed in site/blog/*
`default`, the global layout that `page` extends - I don't recommend using this layout by itself as `page` adds the title from the front matter and the `<article>` element around the content.